backend/app/services/channels/callback.py

```
@@ -271,6 +271,12 @@ async def _remove_emitter(self, task_id: int) -> None:
                     f"[{self._channel_type.value}Callback] Failed to close emitter "
                     f"for task {task_id}"
                 )
+        # Safety cleanup: delete shared streaming content key
+        streaming_content_key = f"channel:streaming_content:{task_id}"
+        try:
+            await cache_manager.delete(streaming_content_key)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)